dev_guide/expanding_persistent_volumes.adoc

```diff
@@ -0,0 +1,118 @@
+[[expanding_persistent_volumes]]
+= Expanding Persistent Volumes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[enabling_expansion_pvc]]
+== Enabling Expansion of Persistent Volume Claims
+
+Volume expansion is a Technology Preview feature, and hence, is not enabled by
+default in your {product-title} {product-version} cluster. There may be other reasons that
+{product-title} administrators wish to enable this feature for certain use
+cases.
+
+[NOTE]
+====
+For more information on Red Hat Technology Preview features support scope,  see
+https://access.redhat.com/support/offerings/techpreview/. 
+====
+
+To allow expansion of persistent volume claims (PVC) by {product-title} users,
+{product-title} administrators must create or update a StorageClass with
+`allowVolumeExpansion` set to `true`. Only PVCs created from that class are
+allowed to expand.
+
+Apart from that, {product-title} administrators must enable the
+`ExpandPersistentVolumes` feature flag and turn on the
+`PersistentVolumeClaimResize` admission controller. Refer to 
+xref:../architecture/additional_concepts/admission_controllers.adoc[Admission Controllers]
```

Comment:
@tmorriso-rh This path should be:

xref:../architecture/additional_concepts/admission_controllers.adoc#architecture-additional-concepts-admission-controllers[Admission Controllers]

It is also good to run asciibinder build and test the links in your preview build